go/vt/utils/flags.go

```diff
@@ -52,15 +53,15 @@ func flagVariants(name string) (underscored, dashed string) {
 func setFlagVar[T any](fs *pflag.FlagSet, p *T, name string, def T, usage string,
 	setFunc func(fs *pflag.FlagSet, p *T, name string, def T, usage string)) {
 
-	underscored, dashed := flagVariants(name)
-	if name == underscored {
+	// underscored, dashed := flagVariants(name)
+	if strings.Contains(name, "_") {
 		fmt.Printf("[WARNING] Please use flag names with dashes instead of underscores, preparing for deprecation of underscores in flag names")
 	}
 
-	setFunc(fs, p, dashed, def, usage)
-	setFunc(fs, p, underscored, def, "")
-	_ = fs.MarkHidden(underscored)
-	_ = fs.MarkDeprecated(underscored, fmt.Sprintf("use %s instead", dashed))
+	setFunc(fs, p, name, def, usage)
+	// setFunc(fs, p, underscored, def, "")
+	// _ = fs.MarkHidden(underscored)
+	// _ = fs.MarkDeprecated(underscored, fmt.Sprintf("use %s instead", dashed))
```

Comment:
TODO: I need to clean this up. I think we can get rid most of the utility functions in here because they don't do anything anymore.

Reply:
We can still use the same normalizer function. The normalizer function is not cobra specific, it's a pflag concept.

Reply:
I just double checked this, and I've already updated the `vtctl` code to use the normalizer function as well: https://github.com/vitessio/vitess/pull/18642/files/afc0d89e2d311170ce8b2b477b541027810790da#diff-a709449242c60f7bc92f0e1cdf204351f5e2eaa776e993b9c1fe75027593d641R95